app/packs/src/decidim/vocdoni/admin/steps/update_census.js

```diff
@@ -0,0 +1,77 @@
+document.addEventListener("DOMContentLoaded", () => {
+  const censusDataElement = document.querySelector(".js-census-data");
+  if (!censusDataElement) {
+    return;
+  }
+
+  let previousUpdateDate = null;
+  const censusDataPath = censusDataElement.dataset.updateCensusUrl;
+
+  const updateCensusInfo = async () => {
+    try {
+      const response = await fetch(censusDataPath, {
+        method: "GET"
+      });
+      if (!response.ok) {
+        throw new Error("Network response was not ok");
+      }
+      const data = await response.json();
+
+      document.getElementById("census-last-updated").textContent = data.info.census_last_updated_at;
+      document.getElementById("census-records-added").textContent = data.info.last_census_update_records_added;
+
+      const usersAwaitingCount = parseInt(data.info.users_awaiting_census.match(/\d+/)[0], 10);
+
+      document.querySelector("[data-users-awaiting-census]").innerHTML = data.info.users_awaiting_census;
+
+      const updateCensusElement = document.getElementById("update-census-link") || document.querySelector(".update-census-span");
+
+      if (updateCensusElement) {
+        if (updateCensusElement.tagName === "A") {
+          if (usersAwaitingCount > 0) {
+            updateCensusElement.classList.remove("disabled");
+          } else {
+            updateCensusElement.classList.add("disabled");
+          }
+        } else if (updateCensusElement.tagName === "SPAN") {
+          if (usersAwaitingCount > 0) {
+            updateCensusElement.classList.remove("disabled");
+            const linkElement = document.createElement("a");
+            linkElement.href = updateCensusElement.parentNode.getAttribute("data-update-url");
+            linkElement.id = "update-census-link";
+            linkElement.className = "button primary alert";
```

Comment:
Let's drop the red color from this button, as it is showing as it's a destructive action and it should be a plain button
```suggestion
            linkElement.className = "button primary";
```